AprilTagDetectionArray[] calibration_data
---
int32[] ids
geometry_msgs/Pose[] poses